app/controllers/bookings_controller.rb

```diff
@@ -0,0 +1,36 @@
+class BookingsController < ApplicationController
+  before_action :set_booking, only: [:show]
+  def index
+    @bookings = policy_scope(Booking).order(created_at: :desc)
```

Comment:
you might need a `@space = Space.find(params[:space_id])` if you are to reference space attributes in the booking index view (which you probably will)

```suggestion
    @space = Space.find(params[:space_id])
    @bookings = policy_scope(Booking).order(created_at: :desc)
```